projects/Apps/common/src/index.ts

```diff
@@ -606,8 +606,7 @@ export interface TrainingEdition extends EditionInterface {}
 
 export interface SpecialEdition extends EditionInterface {
     buttonStyle: SpecialEditionButtonStyles
-    devUri?: string
+    buttonImageUri?: string
```

Comment:
I would say this is required.

Reply:
ok, though maybe we can just fall back to an empty space if the image isn't provided? Though I guess that makes the logic further down more complex